cli/Valet/DnsMasq.php

```diff
@@ -145,6 +148,7 @@ function createTldResolver($tld)
     function updateTld($oldTld, $newTld)
     {
         $this->files->unlink($this->resolverPath.'/'.$oldTld);
+        $this->files->unlink(VALET_HOME_PATH . '/dnsmasq.d/tld-' . $oldTld);
```

Comment:
Should this instead be the same as what we're using to create it?

```php
        $tldConfigFile = $this->dnsmasqUserConfigDir() . 'tld-' . $tld . '.conf';
```

https://github.com/laravel/valet/blob/master/cli/Valet/DnsMasq.php#L120

I'll be honest, though, this is all a bit magical for me.

Reply:
Agreed - consistency here will make maintenance less complicated.